### ISSUE TITLE
Move BackendFrameworkLinuxVersion constant to template catalog

### ISSUE THREAD

**Issue by dgomezc** [Microsoft/WebTemplateStudio#1181]
The BackendFrameworkLinuxVersion property is defined in constants.ts file. This property should be defined in backendframeworks.json file inside the _catalog folder.

**Comment by AndreMaz** [comment]
Not that my opinion counts much but I would suggest to go 1 step further and move all framework related meta-data from `src/client/*` to `templates/_catalog/` dir. When I was adding Moleculer I had to do a lot of hard-coding in `client` dir (see https://github.com/microsoft/WebTemplateStudio/pull/1173/commits/665f217593a6c75c4a8cf2818f0f6108fe474143). This process is error prone and as a result I've missed the `BackendFrameworkLinuxVersion` property https://github.com/microsoft/WebTemplateStudio/pull/1179.

Moving everything into one place would ease and speed up the process of adding new frameworks.

I've also noticed that, at this moment, the [How to add a Framework into WebTS](https://github.com/microsoft/WebTemplateStudio/wiki/Templates#how-to-add-a-framework-into-webts) Wiki page don't explain what files should be updated in order to add a framework to the client, i.e., see it in Wizard
